### DISCUSSION TITLE
Supported image formats

### DISCUSSION THREAD

**Discussion by nick-roux** [yurijmikhalevich/rclip#87] [General]
What image formats are supported? Does it support RAW images?

**Comment by yurijmikhalevich** [comment]
Hi @njrstuff. Currently, rclip supports only jpeg and png images. RAW isn't supported yet. Thank you for expressing your interest. I'll consider adding opt-in RAW support in one of the future versions.

**Comment by yurijmikhalevich** [comment]
@njrstuff, which raw support are you interested the most in?

**Reply by nick-roux** [reply]
Thanks, it will be great if it can support RAW in future.
I'm specifically looking for Canon CR2 support. 

Currently have around 500K images, around 45% jpeg that works brilliantly with rclip and 55% CR2 that is being ignored.

**Reply by yurijmikhalevich** [reply]
@njrstuff, got it. Thank you. Do you have 15-20 minutes to chat about your user flow? I am making a few decisions about the future of rclip, and your input can be valuable. If you are, can you mail me at yurij@mikhalevi.ch?

**Reply by nick-roux** [reply]
We can do that. I'll drop you an email.

**Reply by sylvled** [reply]
> @njrstuff, which raw support are you interested the most in?

Hi.
I'm not Nick, but I would love to have as well RAW support. My best choices would be CR2 and CR3 from Canon.
:)
Thanks and congrats, this tools is amazing :)
Sylvled.

**Reply by yurijmikhalevich** [reply]
Hi @sylvled, thank you for your interest, and I am glad you are loving the tool! I have the feature on my radar.

**Reply by yurijmikhalevich** [reply]
@sylvled, I am trying to understand the direction the rclip project should take, and your input will be helpful. If you have 15 minutes for a call to discuss rclip, send me an email at yurij@mikhalevi.ch, please :)

**Reply by Fethbita** [reply]
Hi there, another voice from me, I am especially interested in Fujifilm RAF files. Although it is probably better to support the main camera brands and their RAW images (Sony ARF, Canon CR2 CR3, Nikon NEF, Fuji RAF, Leica DNG, Olympus ORF, Panasonic RW2).

**Reply by yurijmikhalevich** [reply]
Hi @Fethbita! Thank you for using rclip and for reaching out :) This is on my radar. Will make sure to give attention to the Fuji RAF files, too.

**Comment by yurijmikhalevich** [comment]
@nick-roux, we've released experimental CR2 and ARW RAW support in [rclip 1.11.0](https://github.com/yurijmikhalevich/rclip/releases/tag/v1.11.0). It can be enabled by passing --experimental-raw-support CLI argument. Can you please give it a try and let me know if it works for you?

v1.11.0 isn't available on macOS yet.

**Comment by Fethbita** [comment]
Hi, I know this is closed but should we create a new issue for the formats mentioned in this discussion previously?

**Reply by yurijmikhalevich** [reply]
@Fethbita, the issue is still open here and will remain open until the feature is considered "stable": https://github.com/yurijmikhalevich/rclip/issues/15

You can subscribe there.

I have also reopened this conversation following your comment to track the progress here as well.
